packages/compiler-core/src/transforms/transformElement.ts

```diff
@@ -385,6 +385,13 @@ function resolveSetupReference(name: string, context: TransformContext) {
         `${context.helperString(UNREF)}(${fromMaybeRef})`
       : `$setup[${JSON.stringify(fromMaybeRef)}]`
   }
+
+  const fromProps = checkType(BindingTypes.PROPS)
+  if (fromProps) {
+    return context.inline
+      ? `${context.helperString(UNREF)}(props[${JSON.stringify(fromProps)}])`
+      : `$props[${JSON.stringify(fromProps)}]`
```

Comment:
I have a couple of questions about these two lines.

1. One of them is using `unref` but the other isn't. Is there a reason why we would want to unwrap refs in one case but not the other?
2. Should the first line be using `__props` rather than `props`? I think the latter will only work if there's a `const props = defineProps(...)`, whereas the argument passed to `setup` is called `__props`.

Reply:
Correct. The code has been changed. Thanks.